### PR TITLE
MAISTRA-2723: Update community images URL

### PR DESCRIPTION
**Pull request by jwendell**

### CODE REVIEW

File: manifests-servicemesh/2.1.0/servicemeshoperator.v2.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ metadata:
       The OpenShift Service Mesh Operator enables you to install, configure, and manage an instance of Red Hat OpenShift Service Mesh. OpenShift Service Mesh is based on the open source Istio project.
 
     containerImage: registry.redhat.io/openshift-service-mesh/istio-rhel8-operator:2.1.0
-    createdAt: 2021-10-05T13:11:37EDT
+    createdAt: 2021-11-09T14:40:22EST
```
Comment (rcernich) [inline manifests-servicemesh/2.1.0/servicemeshoperator.v2.1.0.clusterserviceversion.yaml]: We should remove this change.

Comment (jwendell) [reply manifests-servicemesh/2.1.0/servicemeshoperator.v2.1.0.clusterserviceversion.yaml]: fixed

**Review by rcernich — APPROVED** [review]

**Review by dgn — APPROVED** [review]

**Comment by adarshaj** [comment]
After this PR, the new image is causing crash loop back off for `istio-operator` pod, with the following log message:
```json
{
  "level": "error",
  "ts": 1636898507.4141202,
  "logger": "cmd",
  "msg": "",
  "error": "error adding to manager: configuration olm.relatedImage.v1_0.cni must be set",
  "errorVerbose": "configuration olm.relatedImage.v1_0.cni must be set\nerror adding to manager\ngithub.com/maistra/istio-operator/pkg/controller.AddToManager\n\t/builddir/build/BUILD/istio-operator-2.1.0/pkg/controller/controller.go:15\nmain.main\n\t/builddir/build/BUILD/istio-operator-2.1.0/cmd/manager/main.go:168\nruntime.main\n\t/usr/lib/golang/src/runtime/proc.go:255\nruntime.goexit\n\t/usr/lib/golang/src/runtime/asm_amd64.s:1581",
  "stacktrace": "github.com/go-logr/zapr.(*zapLogger).Error\n\t/builddir/build/BUILD/istio-operator-2.1.0/vendor/github.com/go-logr/zapr/zapr.go:132\nmain.main\n\t/builddir/build/BUILD/istio-operator-2.1.0/cmd/manager/main.go:169\nruntime.main\n\t/usr/lib/golang/src/runtime/proc.go:255"
}
```
I'm on OKD 4.8 and installed maistra from the community catalog (i've built my own elasticsearch operator based on redhat's one)
/cc @rcernich @dgn

**Comment by jwendell** [comment]
I think the error is not related to this pr, but it happens because of a change in operator hub. This pr must fix it: https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/436

I'll test it again tomorrow. 

Keep in mind that community images for 2.1 is a work in progress as we speak.

**Pull request merged**